generators/spring-controller/index.js

```diff
@@ -0,0 +1,58 @@
+const path = require('path');
+const Generator = require('yeoman-generator').default;
+
+module.exports = class extends Generator {
```

Comment:
Please reuse others generator convention and integrate it properly (referencing the blueprint in GeneratorsByNamespace for example)